recipes/fmt/all/conanfile.py

```diff
@@ -87,6 +88,10 @@ def package_id(self):
         else:
             del self.info.options.with_fmt_alias
 
+    def validate(self):
+        if self.info.settings.get_safe("compiler.cppstd"):
```

Comment:
broken in conan v1 if header_only is True

Reply:
yes yes, it will be fixed in the next release and beta that's totally fine for now